### ISSUE TITLE
Photo module: add caption color to typography style settings

### ISSUE THREAD

**Issue by gbissland** [beaverbuilder/feature-requests#76]
When a photo caption is selected to be shown, it would be useful (and quicker) to also be able to colour the caption via the settings UI as well.

There are already caption typography options for font family and size, just not color.

![Image 2020-11-09 at 11 46 31 AM](https://user-images.githubusercontent.com/5170216/98486471-4abe2400-2282-11eb-9770-02e5b9df9de6.png)

**Comment by Xirissan** [comment]
Yes please, you would make my life so much easier if you integrate the options : overlay color & hover color like in the column settings <3
